src/lib/configureStore.js

```diff
@@ -15,8 +15,8 @@ import soundsMiddleware from './soundsMiddleware.js'
 const middleware = [errorAlert, loginStatusChecker, thunk, soundsMiddleware]
 if (ENV.ENABLE_REDUX_PERF_LOGGING) middleware.push(perfLogger)
 
-const composeEnhancers =
-  typeof window === 'object' && window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ ? window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__({ name: 'ui', maxAge: 50 }) : compose
+const composeEnhancers = /*
+  typeof window === 'object' && window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ ? window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__({ name: 'ui', maxAge: 50 }) : */ compose
```

Comment:
Instead of commenting it out completely, can you add it to the ENV param and test for it like in the line above with `ENABLE_REDUX_PERF_LOGGING` ?